src/main/java/seedu/task/logic/commands/HelpCommand.java

```diff
@@ -1,26 +1,31 @@
 package seedu.task.logic.commands;
 
-
-import seedu.task.commons.events.ui.ShowHelpEvent;
-import seedu.taskcommons.core.EventsCenter;
-
 /**
  * Format full help instructions for every command for display.
  */
 public class HelpCommand extends Command {
 
-    public static final String COMMAND_WORD = "help";
-
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
-            + "Example: " + COMMAND_WORD;
-
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
-
-    public HelpCommand() {}
-
-    @Override
-    public CommandResult execute() {
-        EventsCenter.getInstance().post(new ShowHelpEvent());
-        return new CommandResult(SHOWING_HELP_MESSAGE);
```

Comment:
U need to fix the compilations errors. 
Somewhere else in the code uses this SHOWING _HELP_MESSAGE. 
Run the junit tests on the whole programme or run ./gradlew clean allTests before pull request.